assembly/src/main/jib/app/stream-registry-config.yaml

```diff
@@ -56,10 +56,19 @@ kafkaStreamsConfig:
     replication.factor: ${STREAM_REGISTRY_KSTREAMS_REPLICATION_FACTOR:-1}
     num.stream.threads: 2
 
-# producerTopic name is the name of the underlying Kafka Topic that would store the Streams created by the application.
-# IMPORTANT: make sure the producerTopic is compacted or retention.ms=-1 so that the created Streams are not lost.
+# eventStoreTopic name is the name of the underlying Kafka Topic that would store the Streams created by the application.
+# IMPORTANT: make sure the eventStoreTopic is compacted or retention.ms=-1 so that the created Streams are not lost.
+# IMPORTANT: Recommended overrides in Prod:
+#   STREAM_REGISTRY_EVENTSTORE_REPLICATION_FACTOR=3
+#   STREAM_REGISTRY_EVENTSTORE_MININSYNCREPLICA=2
 topicsConfig:
-  producerTopic: _streamregistry_streams${STREAM_REGISTRY_PRODUCER_TOPIC_SUFFIX:-_v0.1}
+  eventStoreTopic:
+    name: _streamregistry_streams${STREAM_REGISTRY_EVENTSTORE_TOPIC_SUFFIX:-_v0.1}
+    partitions: ${STREAM_REGISTRY_EVENTSTORE_PARTITIONCOUNT:-1}
+    replicationFactor: ${STREAM_REGISTRY_EVENTSTORE_REPLICATION_FACTOR:-1}
+    properties:
+      min.insync.replicas: ${STREAM_REGISTRY_EVENTSTORE_MININSYNCREPLICA:-1}
+      cleanup.policy: ${STREAM_REGISTRY_EVENTSTORE_CLEANUPPOLICY:-compact}
```

Comment:
Could we do `MIN_ISR` (less to type) and `CLEANUP_POLICY` (easier to read without two P's)?